<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExTrans</class>
 <widget class="QMainWindow" name="ExTrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExTrans</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>绑定端口:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxBind">
          <property name="maximum">
           <number>65000</number>
          </property>
          <property name="value">
           <number>8888</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>目标地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxIp">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>129.0.0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>目标端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxPort">
          <property name="maximum">
           <number>65000</number>
          </property>
          <property name="value">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnUnicast">
        <property name="text">
         <string>单播发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBroadcast">
        <property name="text">
         <string>广播发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actBind"/>
   <addaction name="actDisbind"/>
   <addaction name="separator"/>
   <addaction name="actClean"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actBind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image6.png</normaloff>:/images/Image6.png</iconset>
   </property>
   <property name="text">
    <string>绑定</string>
   </property>
   <property name="toolTip">
    <string>绑定端口</string>
   </property>
  </action>
  <action name="actDisbind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image3.png</normaloff>:/images/Image3.png</iconset>
   </property>
   <property name="text">
    <string>解绑定</string>
   </property>
   <property name="toolTip">
    <string>解除绑定</string>
   </property>
  </action>
  <action name="actClean">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image1.png</normaloff>:/images/Image1.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空文本</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image2.jpg</normaloff>:/images/Image2.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
